infra/migrations/1683613712679_create-index-on-contents-search-terms.js

```diff
@@ -0,0 +1,7 @@
+exports.up = async (pgm) => {
+  await pgm.sql('CREATE EXTENSION unaccent;');
+  await pgm.sql("CREATE INDEX search_term_title ON contents using gin(to_tsvector('portuguese', title));");
+  await pgm.sql("CREATE INDEX search_term_body ON contents using gin(to_tsvector('portuguese', body));");
```

Comment:
```suggestion
  await pgm.sql("CREATE INDEX search_term_contents ON contents using gin(to_tsvector('portuguese', title || ' ' || body));");
```

Será que não fica mais performática a pesquisa se o índice for assim?

Será que esses índices terão impacto significativo no tempo que leva para a inserção de conteúdos?